tests/search/HomeSearch.test.tsx

```diff
@@ -0,0 +1,123 @@
+import HomeSearch from '@src/components/inputs/Search/HomeSearch';
+import { decodeDomainToASCII, lowerCaseDomain } from '@src/utils';
+import { cleanup, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { act } from 'react';
+
+jest.mock('@src/hooks/useArNSRegistryDomains', () => ({
+  useArNSRegistryDomains: jest.fn().mockReturnValue({
```

Comment:
@kunstmusik I think here is where its failing - its not providing the data to the tests